pygdf/_gdf.py

```diff
@@ -466,3 +466,50 @@ def count_nonzero_mask(mask, size):
         libgdf.gdf_count_nonzero_mask(mask_ptr, size, nnz)
 
     return nnz[0]
+
+
+def str_to_gdf_color(s):
+    """Util to convert str to gdf_color type.
+    """
+    return {
+        'green':    libgdf.GDF_GREEN,
+        'blue':     libgdf.GDF_BLUE,
+        'yellow':   libgdf.GDF_YELLOW,
+        'purple':   libgdf.GDF_PURPLE,
+        'cyan':     libgdf.GDF_CYAN,
+        'red':      libgdf.GDF_RED,
+        'white':    libgdf.GDF_WHITE,
+	'dark green': libgdf.GDF_DARK_GREEN,
+	'orange':   libgdf.GDF_ORANGE,
+    }[s.lower()]
```

Comment:
Likewise here, this would be better defined as a global dict, rather than being instantiated each function call.